uniffi/src/lib.rs

```diff
@@ -406,3 +460,12 @@ unsafe impl<V: ViaFfi> ViaFfi for HashMap<String, V> {
         Ok(map)
     }
 }
+
+#[cfg(test)]
+mod test {
+    #[test]
+    fn trybuild_ui_tests() {
+        let t = trybuild::TestCases::new();
+        t.compile_fail("tests/ui/*.rs");
```

Comment:
TIL about `trybuild` and `ui` tests.

I guess any non-failing tests would just be 'tests' not, UI tests.

Reply:
Right.